pkg/agent/watch.go

```diff
@@ -202,6 +204,18 @@ func (e vmEvent) Format(state fmt.State, verb rune) {
 	}
 }
 
+func extractAutoScalingBounds(vm *vmapi.VirtualMachine) *api.ScalingBounds {
```

Comment:
It'd be good to have a comment here explaining what it does and why we have this even though `api.ExtractVmInfo` exists and does some of it